lib/solveEquation/stepThrough.js

```diff
@@ -248,7 +248,7 @@ function solveConstantEquation(equation, debug, steps=[]) {
     throw Error('Unexpected comparator');
   }
 
-  steps = addSimplificationSteps(steps, equation, true, debug);
+  steps = addSimplificationSteps(steps, equation, debug);
```

Comment:
THIS... was driving me crazy. `addSimplificationSteps` only has three parameters, and so the `debug` arg here was always true! That's why mathsteps sometimes console logs a zillion logs when it doesn't need to. A random fix, but a nice one.